site.yml

```diff
@@ -5,6 +5,11 @@
     - consul
 
 - hosts: mesos_masters
```

Comment:
Is it expected that we would change this in a future PR, or now? I'd rather standardise on the approach now so we can just rinse and repeat for each "add-on". 

At a high-level I guess this means setting an environment variable for marathon_enabled (via the bootstrap scripts) and setting its default to true?